detectron2/solver/lr_scheduler.py

```diff
@@ -132,7 +137,7 @@ def get_lr(self) -> List[float]:
 # MultiStepLR with WarmupLR but the current LRScheduler design doesn't allow it.
 
 
-class WarmupMultiStepLR(torch.optim.lr_scheduler._LRScheduler):
```

Comment:
now it should be fine to remove everything below this line. Looks like they are not used any more.

Reply:
@ppwwyyxx There're actually many usage internally (including `_get_warmup_factor_at_iter`), it's a bit hard to remove them.